tests/integration/tile/test_schedule_feature_store.py

```diff
@@ -212,19 +211,18 @@ async def test_schedule_online_feature_store__change_entity_universe(
           select {entity_col_names}, 100.0 as {quote_feature_name} from TEMP_TABLE ORDER BY __FB_TILE_START_DATE_COLUMN ASC limit 2
         )
         """
+    await persistent._update_one(
+        "online_store_compute_query",
+        {"tile_id": tile_id},
+        {"$set": {"sql": new_select_sql}},
     )
-    update_mapping_sql = f"""
-        UPDATE ONLINE_STORE_MAPPING
-        SET SQL_QUERY = '{new_select_sql}'
-        WHERE TILE_ID = '{tile_id}'
-"""
-    await session.execute_query(update_mapping_sql)
 
     tile_online_store_ins = TileScheduleOnlineStore(
         session=session,
         aggregation_id=agg_id,
         job_schedule_ts_str=date_ts_str,
         online_store_table_version_service=online_store_table_version_service,
+        online_store_compute_query_service=online_store_compute_query_service,
```

Comment:
Could we move this to unit tests once all of these operations can be done through mongo?

Reply:
The online store tables will still be stored on the data warehouse, so to test the full operation this will have to be in an integration test.